client/src/modules/stock/exit/exit.js

```diff
@@ -427,6 +429,53 @@ function StockExitController(
     };
   }
 
+  function buildDescription(entityUuid, invoiceUuid) {
+    const glb = {};
+    return PatientService.read(null, { uuid : entityUuid })
+      .then(([patient]) => {
+        if (patient) {
+          glb.patientDescription = patient.display_name.concat(` (${patient.reference})`);
```

Comment:
If we are mixing both $translate and String.concat(), it probably means we can write a better translation template.

I think what you are trying to achieve is to have a translation with something like this:
```
Distribution to the patient JNILES (PA.TBD.123) from ...
```
And your template is
```
Distribution to the patient {{patient}} from...
```

I propose you change your template to be:
```
Distribution to the patient {{patientName}} ({{patientReference}})
```
And then you could just provide the `patientName` and `patientReference` as two different translation keys.